### DISCUSSION TITLE
Trying to use Tasha's code with files sent on 10/28/2024

### DISCUSSION THREAD

**Discussion by Woodyaka** [Woodyaka/Mphys-Project#6] [General]
I am going to be leaving comments here with what I am trying.

**Comment by Woodyaka** [comment]
temp.nc, worked fine once the filepath was adjusted. The output in Open WACCM dataset is the same.

**Comment by Woodyaka** [comment]
![0d3df2d3-7cb7-4d69-a8a7-11367bf80e96](https://github.com/user-attachments/assets/b8868924-06c2-4ef5-943a-50e2cc530078)
This is the figure that was produced on my machine.
![download](https://github.com/user-attachments/assets/fa181f55-a3ee-4f07-a283-55d2ab723e05)
This is the figure on Tasha's github

**Reply by dan800** [reply]
It may be that Tasha used a coarser binning. The morphology certainly looks that same.
